config/initializers/secure_headers.rb

```diff
@@ -13,9 +13,14 @@
     :enforce     => true,
     :default_src => 'self',
     :frame_src   => 'self',
-    :connect_src => 'self ws: wss:',
+    :connect_src => %w(self ws: wss:),
     :style_src   => 'inline self',
-    :script_src  => 'eval inline self',
-    :img_src     => ['self', '*.gravatar.com']
+    :script_src  => %w(eval inline self),
+    :img_src     => %w(self *.gravatar.com)
   }
+  if Rails.env.development? #allow webpack dev server provided assets
+    dev_server = "http://0.0.0.0:#{::Rails.configuration.webpack.dev_server.port}"
```

Comment:
It was using the configuration setting before and I was asked to change it to 0.0.0.0